drivers/scheduler/k8s/k8s.go

```diff
@@ -5047,12 +5048,191 @@ func (k *K8s) CreateCsiSnapsForVolumes(ctx *scheduler.Context, snapClass string)
 					volSnapMap[pvc.Spec.VolumeName] = volSnapshot
 				}
 			}
+
 		}
 	}
 
 	return volSnapMap, nil
 }
 
+// CreateCsiSnapsAndValidate create CSI snapshot for Pure FA volumes and restore them to new PVCs, and validate the content
```

Comment:
Please add the test case IDs. We will integrate to QA team's tool to report the result to the testrail directly in future.
```suggestion
// Tesstrail cases(C58775, C58091) CreateCsiSnapsAndValidate create CSI snapshot for Pure FA volumes and restore them to new PVCs, and validate the content
```